external/fv3fit/fv3fit/emulation/transforms/zhao_carr.py

```diff
@@ -80,6 +96,75 @@ def forward(self, x: TensorDict) -> TensorDict:
         return x
 
 
+def _combine(
+    cloud_before, t_before, t_after, qv_before, qv_after, zero_tendency, zero_cloud,
+):
+    # compute net condensation = Condensation - Evap
+    condensation = qv_before - qv_after
+    cloud_after = cloud_before + condensation
+
+    # apply no change case
+    cloud_out = tf.where(zero_tendency, cloud_before, cloud_after)
+    t_out = tf.where(zero_tendency, t_before, t_after)
+    q_out = tf.where(zero_tendency, qv_before, qv_after)
+
+    # apply zero cloud case
+    # TODO fix these constants
+    latent_heat = 2.51e6
```

Comment:
we do have some constant there already. I guess I preferred copy-paste over the explicit coupling in this instance since it's an approximate value not accounting for ice anyway. I kind of want to gather all things with "zhao carr" in the name into a common directory structure.